pom.xml

```diff
@@ -140,6 +144,19 @@
                 <version>3.12.0</version>
             </dependency>
 
+            <!--
+                RequireUpperBoundDeps:
+                * io.jenkins.plugins:jaxb:2.3.0.1 depends on javax.xml.bind:jaxb-api:2.3.0
+                * com.thoughtworks.xstream:xstream:1.4.17 via org.jenkins-ci.main:jenkins-core:2.303.1
+                    depends on javax.xml.bind:jaxb-api:2.3.1
+                You can remove here if a newer version of jaxb-plugin is released.
+            -->
+            <dependency>
+                <groupId>javax.xml.bind</groupId>
+                <artifactId>jaxb-api</artifactId>
+                <version>2.3.1</version>
+            </dependency>
```

Comment:
1. Why do you placed this in dependency managment rather than in to dependencies?
2. As far as I can tell we should use api plugins 
https://plugins.jenkins.io/jaxb/

Reply:
> 2. As far as I can tell we should use api plugins https://plugins.jenkins.io/jaxb/

We does.
Have a look on L.116-L.119.

L.116-L.119 is the purpose of this pull request. 
L.147-L.158 is just a workaround for `RequireUpperBoundDeps`.

Reply:
> 1. Why do you placed this in dependency managment rather than in to dependencies?

`RequireUpperBoundDeps` happens like this:

```
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for javax.xml.bind:jaxb-api:2.3.0 paths to dependency are:
+-org.jenkins-ci.plugins:crowd2:3.1.1-SNAPSHOT
  +-io.jenkins.plugins:jaxb:2.3.0.1
    +-javax.xml.bind:jaxb-api:2.3.0
and
+-org.jenkins-ci.plugins:crowd2:3.1.1-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.303.1 [provided]
    +-com.thoughtworks.xstream:xstream:1.4.17 [provided] (managed) <-- com.thoughtworks.xstream:xstream:1.4.17 [provided]
      +-javax.xml.bind:jaxb-api:2.3.1 [provided]
]
```

So what all we need is to let maven know it should use jaxb-api-2.3.1 instead of jaxb-2.3.0. That's why it's described in `<dependencyManagement>.
(Actually, that fixes the error. Please try that if you're interested in that.)

Describing in `<dependencyManagement>` instead of the top `<dependencies>` has following good points:

* crowd2.hpi won't contain jaxb-api.jar. crowd2.hpi get larger if you specify the dependency to jaxb-api in the top `<dependency>`.
* Other developers can get "Oh, this means crowd2-plugin itself don't directly depend to jaxb-api, but it's just to fix dependency version issues."

Details about `RequireUpperBoundDeps` is described here:

* https://www.jenkins.io/doc/developer/plugin-development/dependency-management/ (Unfortunately, it doesn't describe how to fix that).
* https://github.com/jenkins-infra/docker-confluence-data/blob/main/content/JENKINS/How-to-fix-RequireUpperBoundDeps.html (In html, and really hard to read on github...)

Reply:
Another option is specifying `<exclude>` for `io.jenkins.plugins:jaxb` just like `httpcomponents` for `crowd-integration-client-rest`: https://github.com/jenkinsci/crowd2-plugin/blob/crowd2-2.2.0/pom.xml#L97 .
Please let me know if you prefer that.